pricegraph/src/orderbook/order.rs

```diff
@@ -113,5 +187,5 @@ fn is_unbounded(element: &Element) -> bool {
 /// Calculates an effective price as a `f64` from a price fraction.
 fn as_effective_sell_price(price: &Price) -> f64 {
     const FEE_FACTOR: f64 = 1.0 / 0.999;
-    FEE_FACTOR * (price.denominator as f64) / (price.numerator as f64)
+    FEE_FACTOR * (price.numerator as f64) / (price.denominator as f64)
```

Comment:
🤦 Who is reviewing this code? 😉

Reply:
I only noticed it because I got to a point where I was removing a way too many negative cycles, then I realized that I had the prices the wrong way :facepalm: